styles.css

```diff
@@ -69,3 +110,135 @@
 .contact-info-button {
   margin-right: 30px;
 }
+
+/* Recent works */
+.recent-works {
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.works-cards-container {
+  width: 90%;
```

Comment:
- [ ] You did an amazing job styling the cards and please stick as much as you can following (e.g., font, colors, images, tests, margins) the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

Here you have a screenshot of your card:

![image](https://user-images.githubusercontent.com/74506933/158911666-2cfe93c2-04c2-44c3-ab75-b74e970226f7.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/158911691-2d77b732-b21c-4887-a65e-ebd51fc6428e.png)